examples/point_cloud/main.cpp

```diff
@@ -15,8 +15,27 @@ static char const* p_xyz_frame_output_file_name = "XYZFrame.bin";
 static const char kUsage[] =
 R"(PointCloud.
     kUsage:
-      PointCloud --input_file=<Input depth file> --ccb_file=<ccb_file> --mode=<mode> --output_folder=<Output_folder>
-      PointCloud (-h | --help) 
+      PointCloud (-h | --help)
+      first-frame [-input_file | --input_file <Input depth file>] [-ccb_file | --ccb_file <ccb_file>] [-mode | --mode <mode>] [-output_folder | --output_folder <Output_folder>]
```

Comment:
"first-frame" should not be written in this menu

Reply:
Solved